cdist/__init__.py

```diff
@@ -56,6 +56,20 @@ class UnresolvableRequirementsError(cdist.Error):
     pass
 
 
+class CdistBetaRequired(cdist.Error):
+    """Beta functionality is used but beta is not enabled"""
+
+    def __init__(self, command, arg):
+        self.command = command
+        self.arg = arg
+
+    def __str__(self):
+        err_msg = ("\'{}\' argument of \'{}\' command is beta, but beta is "
+                   "not enabled. If you want to use it please enable beta "
+                   "functionalities.")
```

Comment:
... "using the --enable-beta command line flag" -- do not let the user guess or force her to lookup how to do it, if a simple hint can avoid that